addons/stock/wizard/stock_picking_return_views.xml

```diff
@@ -29,6 +29,7 @@
                     </tree>
                 </field>
                 <group>
+                    <field name="location_id" groups="!stock.group_stock_multi_locations" invisible="1"/>
```

Comment:
That does not seem correct to me; if I am not wrong, groups="..." is replaced by invisible="1".
So, the field is not removed from the view, it's just invisible. So, adding a second time the same field
in the view should have no positive impact.

Reply:
Cancel this PR, see my review on the task.